### DISCUSSION TITLE
no sound after upgrade kernel to 6.6.39 from 6.6.34 KDE version hardinfo and pavucontrol show dummy output

### DISCUSSION THREAD

**Discussion by constantius24** [mocaccinoOS/mocaccino#135] [General] [answered]
no sound after upgrade kernel to 6.6.39 from 6.6.34 ( KDE version)  hardinfo and pavucontrol show dummy output
reinstall kernel help nothing. Maybe should I install firmware but which one? driver for Realtek  ALC3234

**Comment by msdobrescu** [comment]
Hello, in my case was no kernel problem.
The source of this is here: https://pipewire.pages.freedesktop.org/wireplumber/daemon/configuration/migration.html
In my case, removing the old configs fixed it (I did an oscheck after by luet oscheck --reinstall).

**Comment by constantius24** [comment]
luet oscheck --reinstall - this crash my system - i cant login - rescue mode does not work as well

**Reply by msdobrescu** [reply]
This checks the existing files against the files that are contained by the installed packages, if some are missing, it simply reinstall the packages if you confirm to.
However, in the case of system breakage, I usually chroot and run luet install for some packages I assess are missing for some reason (like kernel and the desired desktop, maybe a kernet-updater or a grub reinstall). In the worst case, if bash dependencies are missing, you can copy the necessary files from the ISO image mounted for chrooting.
This is general, I can't actually help you as I have no idea of what kind of crash you have, but this fixed my system in the worst scenarios.

**Reply by constantius24** [reply, marked as answer]
I did fresh install ( kernel 6.6.32 ,) and then luet upgrade ( kernel 6.6.39) and everything works

**Reply by msdobrescu** [reply]
Glad to hear you fixed it! I prefer to fix it as described as luet helps a lot to do that.
